### ISSUE TITLE
Unsupported locale

### ISSUE THREAD

**Issue by brasilikum** [benhutchins/docker-taiga-example#4]
Running the advanced config results in an error:

```
 ---> Running in d06b26d0ec24
Traceback (most recent call last):
  File "/usr/local/bin/pip", line 11, in <module>
    sys.exit(main())
  File "/usr/local/lib/python3.4/site-packages/pip/__init__.py", line 215, in main
    locale.setlocale(locale.LC_ALL, '')
  File "/usr/local/lib/python3.4/locale.py", line 592, in setlocale
    return _setlocale(category, locale)
locale.Error: unsupported locale setting
ERROR: Service 'taiga' failed to build: The command '/bin/sh -c pip install --no-cache-dir       taiga-contrib-slack       taiga-contrib-ldap-auth' returned a non-zero code: 1
```

**Comment by josequaresma** [comment]
Hi. I had the same issue @brasilikum . adding `ENV LANG C` to the Dockerfile before the `RUN pip` command did the trick

**Comment by benhutchins** [comment]
Thanks @josequaresma, I'll make a PR to fix this soon.

**Comment by benhutchins** [comment]
Should be fixed with benhutchins/docker-teika@842efe1
